app/observers/aaib_report_observers_registry.rb

```diff
@@ -1,5 +1,7 @@
-require "email_alert_exporter"
 require "formatters/aaib_report_publication_alert_formatter"
+require "formatters/aaib_report_indexable_formatter"
+require "markdown_attachment_processor"
```

Comment:
Good catch. I'll move the require for `rummager_indexer` into the superclass, but I'll leave the require for `markdown_attachment_processer` in the subclasses as that's where the class is referenced.